style.css

```diff
@@ -0,0 +1,15 @@
+.show {
```

Comment:
almost all these classes you could have used bootstrap classes
 d-none, d-block, or text-danger